.github/workflows/ci.yaml

```diff
@@ -11,6 +11,17 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
+  latest-sdk-versions:
```

Comment:
Using an entire no-op job just to avoid the variable being used twice is a bit much. Maybe we can just use `env` at the top-level of this file and use `${{ env.NAME_OF_VAR }}` as needed within? I know it may seem side-effecting to alter the environment variables of these runs, but should be harmless (don't name them something too common). That's better than a no-op job IMO.

Reply:
+1 to that. It's so lame github's parser doesn't allow normal substitutions

Reply:
That was my initial attempt, but GHA doesn't allow accessing the `env` context from `jobs.<job_id>.with` ([see ref](https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability)).

Using repo-level variables would be a possibility, but I don't think we should go there. Unless I'm mistaken, that would make builds non-reproductible.